theme/src/components/skip-link.js

```diff
@@ -0,0 +1,38 @@
+import {Link} from '@primer/components'
+import styled from 'styled-components'
+import React from 'react'
+
+function SkipLinkBase({...rest}) {
+  return (
+    <Link
+      {...rest}
+      backgroundColor="blue.6"
+      color="white"
+      p={3}
+      href="#skip-nav"
+      fontSize={14}
+    >
+      Skip to content
+    </Link>
+  )
+}
+
+const SkipLink = styled(SkipLinkBase)`
+  z-index: 20;
+  width: auto;
+  height: auto;
+  clip: auto;
+  position: absolute;
+  overflow: hidden;
+
+  &:not(:focus) {
+    clip: rect(1px, 1px, 1px, 1px);
```

Comment:
It might be worth leaving a comment here explaining that all these rules are used to visually hide the element.

Reply:
Yep, makes sense! 
Also, I was thinking it might be nice to add a few utility components to primer/components. Something like 
`<SrOnly />` and `<VisibleOnFocus />` because I think they might be applicable in other scenarios. @emplums, what do you think?